app/views/devise/sessions/new.html.slim

```diff
@@ -1,25 +1,28 @@
-h2
-  | Log in
-
-= render 'shared/notifications'
```

Comment:
Error handling has not been discussed and needs to be defined before being implemented.
For March it might be ok, but long term error messages and notifications must be aligned with the rest of the products.
How should we proceed?